concent_api/core/constants.py

```diff
@@ -90,3 +90,6 @@ class VerificationResult(IntEnum):
 
 # Defines a string that is returned from Bankster to Core views to indicate timestamp error case.
 BANKSTER_TIMESTAMP_ERROR = 'timestamp_error'
+
+
+ZERO_SIGNATURE = b'\x00' * 65
```

Comment:
Isn't this available in golem-messages? If yes, maybe it would be better to import it from there.

Reply:
What is this signature?

Reply:
seems strange for me... but if is is so - :+1: